1.4/Defs/TradeValidationDefs.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+	<DynamicTradeInterface.Defs.TradeValidationDef>
+		<defName>DF_InsufficientSilver</defName>
+		<validation>DynamicTradeInterface.UserInterface.TradeValidation.InsufficientSilver:Validate</validation>
+		<text>ConfirmTraderShortFunds</text>
```

Comment:
what does "text" mean? Translation key?

Reply:
I haven't used keys directly in xml anywhere else, only for hard-coded values since XML values can be patched.

Reply:
Yes, text is intended to be a translation key. I think it is the best way to go, as the existing confirmation dialogs we are targetting are already using translatable strings for the text.

Reply:
This field has also been renamed to `textKey` to show explicitly that it is a translation key.